x-pack/plugins/lists/server/services/lists/get_list_template.test.ts

```diff
@@ -24,8 +24,15 @@ describe('get_list_template', () => {
     const template = getListTemplate('some_index');
     expect(template).toEqual({
       index_patterns: ['some_index-*'],
-      mappings: { listMappings: {} },
```

Comment:
This looks strange; is `listMappings` really supposed to be a key in this template?